app/uk/gov/hmrc/economiccrimelevyregistration/views/RegistrationReceivedView.scala.html

```diff
@@ -26,15 +27,15 @@
 @(firstContactEmailAddress: String, secondContactEmailAddress: Option[String], liabilityYear: Option[LiabilityYear], registeringForCurrentYear: Boolean)(implicit request: Request[_], messages: Messages)
 
 @getPreviousYearContent() = {
-    <p class="govuk-body">@messages("registration.received.p7.previous.p1")</p>
+    <p class="govuk-body">@messages("registration.received.p7.previous.p1", liabilityYear.get.asString, liabilityYear.get.followingYear)</p>
     <p class="govuk-body">@messages("registration.received.p7.previous.p2")</p>
 }
 @getCurrentYearContent() = {
-    <p class="govuk-body">@messages("registration.received.p7.current")</p>
+    <p class="govuk-body">@messages("registration.received.p7.current", formatLocalDate(EclTaxYear.dueDate))</p>
 }
 @getBothYearsContent() = {
-    <p class="govuk-body">@messages("registration.received.p7.both.years.p1")</p>
-    <p class="govuk-body">@messages("registration.received.p7.both.years.p2")</p>
+    <p class="govuk-body">@messages("registration.received.p7.both.years.p1", formatLocalDate(EclTaxYear.dueDate))</p>
```

Comment:
The EclTaxYear.dueDate only shows this year's due date. 30th of September is currently what it says, which is incorrect for the 2024-2025 ECL tax year.